tests/test-1.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from "@playwright/test";
+
+test("test", async ({ page }) => {
```

Comment:
ang name sa imung test dapat related xa objective nimu. 

example:

test("should be able to login and create a new topic and note")